src/components/macro/macro.component.scss

```diff
@@ -98,7 +101,7 @@ h1 {
 }
 
 .add-new__action-container {
-    border-top: 1px solid #ddd;
+    overflow: hidden;
```

Comment:
Could you also add `flex-shrink: 0;` ? 
When a macro is selected for edit the buttons at the bottom disappear for me, but it solves the problem.